### PR TITLE
PEP8 and pyflakes fixes

### PR DESCRIPTION
**Pull request by jaingaurav**

### CODE REVIEW

**Comment by Lacrymology** [comment]
:+1:

**Comment by deejay1** [comment]
:+1:

File: src/collectors/monit/monit.py [hunk 1]
```diff
@@ -106,7 +106,7 @@ def collect(self):
                                 metrics["%s.memory_total.%s_usage" % (
                                     name, unit)] = (
                                     diamond.convertor.binary.convert(
-                                        value=mem,
+                                        value=mem_total,
```
Comment (mattrobenolt) [inline src/collectors/monit/monit.py]: Is this change intentional?

File: src/diamond/collector.py [hunk 2]
```diff
@@ -233,7 +233,7 @@ def process_config(self):
                 and self.config.get('metrics_blacklist', None)):
             raise DiamondException(
                 'Both metrics_whitelist and metrics_blacklist specified ' +
-                'in file %s' % configfile)
+                'in file %s' % self.configfile)
```
Comment (mattrobenolt) [inline src/diamond/collector.py]: Is this intentional?

**Comment by mattrobenolt** [comment]
Other than those 2 comments, this looks fine.

File: src/collectors/monit/monit.py (hunk 1 above)
Comment (jaingaurav) [inline src/collectors/monit/monit.py]: Yes it was a subtle bug I discovered. Probably should've been broken up into a separate commit

Comment (mattrobenolt) [inline src/collectors/monit/monit.py]: I'd prefer fixing those bugs in a separate PR. :)

File: src/diamond/collector.py (hunk 2 above)
Comment (jaingaurav) [inline src/diamond/collector.py]: Yeah same here, should've broken it out

**Comment by mattrobenolt** [comment]
:cake:
